### ISSUE TITLE
Cannot change configuration to allow bare ddl

### ISSUE THREAD

**Issue by brassel** [edgedb/edgedb#4672]
- EdgeDB Version:
edgedb --version
EdgeDB CLI 2.1.0+527e309

edgedb instance list
2.6+db69671 

- OS Version:
Mac OS 12.6 (21G115)

Steps to Reproduce:

1. create new project and initialize instance.
2. notice that bare ddl is not allowed, e.g., by using the repl in `edgedb ui`
3. execute   `edgedb configure set allow_bare_ddl AlwaysAllow ` to get `OK: CONFIGURE INSTANCE`
4. notice that bare ddl is still not allowed
5. (optional step) create a new database in the edgedb ui and notice that bare ddl is allowed there by default.

**Comment by msullivan** [comment]
Running `configure current database set allow_bare_ddl := cfg::AllowBareDDL.AlwaysAllow` looks like it works.

**Comment by elprans** [comment]
Yeah, IMO we should consider adding `edgedb instance configure` and `edgedb database configure` (and making `edgedb configure` an alias to the former).

**Comment by brassel** [comment]
Ah, yes, I see, thank you for the information! Adding to my confusion was that `allow_user_specified_id` would work with just `edgedb configure`. Now I got the picture, thank you!
From my side the issue could be closed. But I leave it open , for your discussion about the maybe changes to the commands.
